src/custom/api/gnosisProtocol/api.ts

```diff
@@ -193,13 +194,31 @@ const UNHANDLED_METADATA_ERROR: MetadataApiErrorObject = {
   description: MetadataApiErrorCodeDetails.UNHANDLED_GET_ERROR,
 }
 
-async function _handleQuoteResponse(response: Response) {
+async function _handleQuoteResponse(response: Response, params?: FeeQuoteParams) {
   if (!response.ok) {
     const errorObj: ApiErrorObject = await response.json()
 
     // we need to map the backend error codes to match our own for quotes
     const mappedError = mapOperatorErrorToQuoteError(errorObj)
-    throw new QuoteError(mappedError)
+    const quoteError = new QuoteError(mappedError)
+
+    if (params) {
+      const { sellToken, buyToken } = params
+
+      const sentryError = new Error()
+      Object.assign(sentryError, quoteError, {
+        message: `Error querying fee from API - sellToken: ${sellToken}, buyToken: ${buyToken}`,
+        name: 'FeeErrorObject',
+      })
+
+      // report to sentry
+      Sentry.captureException(sentryError, {
+        tags: { errorType: 'getFeeQuote' },
+        contexts: { params },
+      })
+    }
```

Comment:
I would expect them to make this fail safe. If not we can always wrap it on our own util, but Im pretty sure this is what they would also do (try-catch logic in case the network is down)

Maybe is sth that is better to test, or review whats the code inside there. Otherwise we will be wrapping in try-catch all the time for not big reason